cmd/es-index-cleaner/app/flags.go

```diff
@@ -44,18 +46,23 @@ type Config struct {
 
 // AddFlags adds flags for TLS to the FlagSet.
 func (c *Config) AddFlags(flags *flag.FlagSet) {
-	flags.String(indexPrefix, "", "Index prefix")
+	flags.String(deprecatedIndexPrefix, "", deprecatedIndexPrefixWarning+" see --"+indexPrefix)
 	flags.Bool(archive, false, "Whether to remove archive indices. It works only for rollover")
 	flags.Bool(rollover, false, "Whether to remove indices created by rollover")
 	flags.Int(timeout, 120, "Number of seconds to wait for master node response")
 	flags.String(indexDateSeparator, "-", "Index date separator")
 	flags.String(username, "", "The username required by storage")
 	flags.String(password, "", "The password required by storage")
+	flags.String(indexPrefix, "", "Index prefix")
 }
 
 // InitFromViper initializes config from viper.Viper.
 func (c *Config) InitFromViper(v *viper.Viper) {
 	c.IndexPrefix = v.GetString(indexPrefix)
+	if c.IndexPrefix == "" {
+		// try with deprecated flag
+		c.IndexPrefix = v.GetString(deprecatedIndexPrefix)
```

Comment:
this should log a warning about using deprecated flag

Reply:
solved